cmd/agent/agent.go

```diff
@@ -48,9 +49,11 @@ func loop(c *cli.Context) error {
 		hostname, _ = os.Hostname()
 	}
 
+	platform := runtime.GOOS + "/" + runtime.GOARCH
```

Comment:
on agent start can we print out some info ... so it's nice for debuging ... what ID, platform, hostname ,etc ...

Reply:
done